src/Microsoft.DotNet.Interactive.Documents/ParserServer/NotebookParserServer.cs

```diff
@@ -33,6 +33,7 @@ public NotebookParserServer(TextReader input, TextWriter output)
         new("sql", languageName: "SQL"),
         new("kql", languageName: "KQL"),
         new("mermaid", languageName: "Mermaid"),
+        new("httpRequest", languageName: "HTTP Request"),
```

Comment:
Double check in the kennel itself, but the language name should be "http".

Reply:
correct''!